src/Glpi/Asset/CustomField/PlaceholderType.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Asset\CustomField;
+
+use Glpi\Application\View\TemplateRenderer;
+
+class PlaceholderType extends AbstractType
```

Comment:
I discussed about it with @orthagh and we agree on the fact that having we should not have to store a specific `CustomField` for this. Indeed, we plan to implement a feature to reorder the fields displayed in the custom asset form, and we should probably store the presence of a placeholder in a `form_layout`/`fields_display`/`whatever` field of the asset definition.

Removing the `Placeholder` type would also simplify multiple pieces of code in the current PR.